nixos/modules/services/security/tor.nix

```diff
@@ -705,7 +705,6 @@ in
 
         # Translated from the upstream contrib/dist/tor.service.in
         preStart = ''
-          install -o tor -g tor -d ${torDirectory}/onion ${torRunDirectory}
           ${pkgs.tor}/bin/tor -f ${torRcFile} --verify-config
         '';
```

Comment:
`serviceConfig.ExecStartPre` can be then used. This saves one derivation because no script has to be created. Also it generates more readable systemd units.